--- abuild
+++ abuild
@@ -2214,11 +2214,12 @@
 done
 shift $(( $OPTIND - 1 ))
 
+# Fully allow us to run as root, for the hax!
 # check so we are not root
-if [ $(id -u) -eq 0 ] && [ -z "$FAKEROOTKEY" ]; then
-	[ -z "$forceroot" ] && die "Do not run abuild as root"
-	FAKEROOT=
-fi
+# if [ $(id -u) -eq 0 ] && [ -z "$FAKEROOTKEY" ]; then
+#     [ -z "$forceroot" ] && die "Do not run abuild as root"
+#     FAKEROOT=
+# fi
 
 # find startdir
 [ -f "$APKBUILD" ] || die "Could not find $APKBUILD (PWD=$PWD)"
