diff --git a/alpine-php/builder/Dockerfile.tmpl b/alpine-php/builder/Dockerfile.tmpl
index a789060..13f0b0e 100644
--- a/alpine-php/builder/Dockerfile.tmpl
+++ b/alpine-php/builder/Dockerfile.tmpl
@@ -25,7 +25,7 @@ LABEL org.label-schema.schema-version="1.0" \
       org.label-schema.distribution="Alpine Linux" \
       org.label-schema.distribution-version="3.12" \
       info.humanitarianresponse.php=$VERSION \
-      info.humanitarianresponse.php.modules="bcmath bz2 calendar ctype curl dom exif fileinfo fpm ftp gd gettext gmp iconv igbinary imagick imap intl json mcrypt memcached opcache openssl pdo pdo_mysql phar posix redis shmop soap sysvmsg sysvsem sysvshm simplexml sockets wddx xml xmlreader xmlwriter xsl zip zlib" \
+      info.humanitarianresponse.php.modules="bcmath bz2 calendar ctype curl dom exif fileinfo fpm ftp gd gettext gmp iconv igbinary imagick imap intl json mcrypt memcached opcache openssl pdo pdo_mysql phar posix redis shmop soap sysvmsg sysvsem sysvshm simplexml sockets xml xmlreader xmlwriter xsl zip zlib" \
       info.humanitarianresponse.php.sapi="cli" \
       info.humanitarianresponse.composer="1.10.19" \
       info.humanitarianresponse.drush="8.4.6" \
@@ -41,7 +41,7 @@ ENV DRUSH_VERSION=8 \
     DRUSH_RELEASE=8.x-dev
 
 COPY drushrc.php /drushrc.php
-COPY etc/php7/php.ini /tmp/php.ini
+COPY etc/php8/php.ini /tmp/php.ini
 
 RUN npm install --global npm@latest && \
     npm install --global \
@@ -61,78 +61,59 @@ RUN npm install --global npm@latest && \
       rsync \
       tar \
       wget \
-      php7-bcmath \
-      php7-bz2 \
-      php7-calendar \
-      php7-ctype \
-      php7-curl \
-      php7-dom \
-      php7-exif \
-      php7-iconv \
-      php7-fileinfo \
-      php7-ftp \
-      php7-gd \
-      php7-gettext \
-      php7-gmp \
-      php7-iconv \
-      php7-imagick \
-      php7-imap \
-      php7-intl \
-      php7-json \
-      php7-mbstring \
-      php7-mcrypt \
-      php7-memcached \
-      php7-mysqli \
-      php7-opcache \
-      php7-openssl \
-      php7-pdo \
-      php7-pdo_mysql \
-      php7-phar \
-      php7-posix \
-      php7-redis \
-      php7-shmop \
-      php7-sysvmsg \
-      php7-sysvsem \
-      php7-sysvshm \
-      php7-simplexml \
-      php7-soap \
-      php7-sockets \
-      php7-tokenizer \
-      php7-wddx \
-      php7-xml \
-      php7-xmlreader \
-      php7-xmlwriter \
-      php7-xsl \
-      php7-zip \
-      php7-zlib && \
+      php8-bcmath \
+      php8-bz2 \
+      php8-calendar \
+      php8-ctype \
+      php8-curl \
+      php8-dom \
+      php8-exif \
+      php8-iconv \
+      php8-fileinfo \
+      php8-ftp \
+      php8-gd \
+      php8-gettext \
+      php8-gmp \
+      php8-iconv \
+      php8-imap \
+      php8-intl \
+      php8-json \
+      php8-mbstring \
+      php8-mysqli \
+      php8-opcache \
+      php8-openssl \
+      php8-pdo \
+      php8-pdo_mysql \
+      php8-phar \
+      php8-posix \
+      php8-redis \
+      php8-shmop \
+      php8-sysvmsg \
+      php8-sysvsem \
+      php8-sysvshm \
+      php8-simplexml \
+      php8-soap \
+      php8-sockets \
+      php8-tokenizer \
+      php8-xml \
+      php8-xmlreader \
+      php8-xmlwriter \
+      php8-xsl \
+      php8-zip \
+      php8-zlib \
+      php8-pecl-imagick \
+      php8-pecl-mcrypt \
+      php8-pecl-memcached && \
     rm -rf /var/cache/apk/* && \
     ln -sf /usr/bin/python3 /usr/bin/python && \
     cd /tmp && \
-    mv /tmp/php.ini /etc/php7/php.ini && \
-    ln -sf /usr/bin/php7 /usr/bin/php && \
+    mv /tmp/php.ini /etc/php8/php.ini && \
+    ln -sf /usr/bin/php8 /usr/bin/php && \
     COMPOSER_HASH=$(curl -sS https://composer.github.io/installer.sig) && \
     php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');" && \
     php -r "if (hash_file('SHA384', 'composer-setup.php') === '$COMPOSER_HASH') { echo 'Installer verified'; } else { echo 'Installer corrupt'; unlink('composer-setup.php'); } echo PHP_EOL;" && \
     php composer-setup.php --install-dir=/usr/bin --filename=composer && \
     php -r "unlink('composer-setup.php');" && \
-    # Downgrade composer to v1.
-    composer self-update --1 && \
-    # Install drush and symlink it somewhere useful.
-    COMPOSER_HOME=/usr/local/drush$DRUSH_VERSION \
-      composer global require drush/drush:$DRUSH_RELEASE && \
-    cd / && \
-    ln -sf /usr/local/drush$DRUSH_VERSION/vendor/bin/drush /usr/bin/drush && \
-    drush status && \
-    # Install registry_rebuild to the appuser's homedir.
-    drush -y @none dl --destination=/home/appuser/.drush/commands registry_rebuild-7.x && \
-    # Ensure the appuser's drush directory is writable.
-    chown -R appuser:appuser /home/appuser/.drush && \
-    # Symlink the appuser's drush module to root's.
-    mkdir -p /root/.drush && \
-    ln -s /home/appuser/.drush/commands /root/.drush/commands && \
-    # Add our default drushrc.php file.
-    mkdir -p /etc/drush && \
-    mv /drushrc.php /etc/drush/drushrc.php && \
     # Add dependencies for capsper and phantom, but do NOT install them. npm will fetch a newer one anyway.
     apk add --update-cache \
       fontconfig \
diff --git a/alpine-php/builder/etc/php7/php.ini b/alpine-php/builder/etc/php8/php.ini
similarity index 100%
rename from alpine-php/builder/etc/php7/php.ini
rename to alpine-php/builder/etc/php8/php.ini
